tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/statements/matchstmt/MatchStatementSyntaxErrorsTest.java

```diff
@@ -58,7 +58,7 @@ public void testSyntaxErrors() {
         BAssertUtil.validateError(result, ++i, "missing close brace token", 18, 1);
         BAssertUtil.validateError(result, ++i, "missing colon token", 21, 11);
         BAssertUtil.validateError(result, ++i, "missing identifier", 21, 11);
-        BAssertUtil.validateError(result, ++i, "unsupported match pattern", 21, 11);
+        BAssertUtil.validateError(result, ++i, "variable '$missingNode$_1' should be declared as constant", 21, 11);
```

Comment:
Is there any way to get rid of this  `$missingNode$_1` in the error msg?

Reply:
As we discussed earlier, for now, it is fine. But, should be addressed later. Issue created. #29168